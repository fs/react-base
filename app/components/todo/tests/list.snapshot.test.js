import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TodoList from '../list';

describe('TodoList', () => {
  let props;
  const renderComponent = () => shallow(<TodoList {...props} />);

  beforeEach(() => {
    props = {
      deleteTodo: () => {},
      todos: [
        { id: 1, isComplete: false, name: 'Something to do 1' },
        { id: 2, isComplete: false, name: 'Something to do 2' },
      ],
      updateTodo: () => {},
    };
  });

  it('renders correctly', () => {
    expect(toJson(renderComponent())).toMatchSnapshot();
  });
});
