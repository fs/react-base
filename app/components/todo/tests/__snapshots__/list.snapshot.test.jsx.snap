// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList renders correctly 1`] = `
<ListGroup
  bsClass="list-group"
>
  <TodoItem
    deleteTodo={[Function]}
    key="1"
    todo={
      Object {
        "id": 1,
        "isComplete": false,
        "name": "Something to do 1",
      }
    }
    updateTodo={[Function]}
  />
  <TodoItem
    deleteTodo={[Function]}
    key="2"
    todo={
      Object {
        "id": 2,
        "isComplete": false,
        "name": "Something to do 2",
      }
    }
    updateTodo={[Function]}
  />
</ListGroup>
`;
